src/API/.envrc.template

```diff
@@ -7,4 +7,4 @@ export PORT=7000
 export AUTH0_ISSUER_URL='https://dev-326tk4zu.us.auth0.com/'
 export AUTH0_AUDIENCE='https://www.vectoratlas.org'
 export TOKEN_KEY='<token_key>'
-export AZURE_STORAGE_CONNECTION_STRING='<connection_string>'
\ No newline at end of file
+export AZURE_STORAGE_CONNECTION_STRING='DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;'
```

Comment:
Uh-oh :-)

Reply:
Oh wait, this is for the local one?

Reply:
Yes, quite right - this is only for the emulator and is publicly available on the Microsoft site anyway. There's nothing secret about this value.